e2e/test-component/scenarios/embedding-sdk/native-question.cy.spec.tsx

```diff
@@ -63,4 +66,41 @@ describeEE("scenarios > embedding-sdk > interactive-question > native", () => {
         .should("have.text", String(ORDERS_ID));
     });
   });
+
+  it("supports passing sql parameters to static questions", () => {
```

Comment:
(I'll add more test cases for rendering questions + updating the viz selector after I'm done with fixing the CI)